keras_cv/utils/visualize.py

```diff
@@ -0,0 +1,67 @@
+import tensorflow as tf
```

Comment:
Lets actually make a new directory.  `keras_cv/visualization`.  Then this file can go in `keras_cv/visualization/segmentation_mask.py` and __init__.py can import the API symbols from that file.

Reply:
done